### ISSUE TITLE
occur error when training image

### ISSUE THREAD

**Issue by nguyenvannhut1993** [google-research/uda#47]
I0927 03:16:41.333719 140651592308160 tpu_estimator.py:2160] examples/sec: 186.336
INFO:tensorflow:global_step/sec: 5.45765
I0927 03:16:41.516547 140651592308160 tpu_estimator.py:2159] global_step/sec: 5.45765
INFO:tensorflow:examples/sec: 174.645
I0927 03:16:41.516915 140651592308160 tpu_estimator.py:2160] examples/sec: 174.645
ERROR:tensorflow:Error recorded from training_loop: 2 root error(s) found.
  (0) Data loss: truncated record at 49210074
         [[node IteratorGetNext (defined at main.py:544) ]]
         [[IteratorGetNext/_1677]]
  (1) Data loss: truncated record at 49210074
         [[node IteratorGetNext (defined at main.py:544) ]]
0 successful operations.
0 derived errors ignored.
Original stack trace for u'IteratorGetNext':
  File "main.py", line 588, in <module>
    tf.app.run()
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/platform/app.py", line 40, in run
    _run(main=main, argv=argv, flags_parser=_parse_flags_tolerate_undef)
  File "/usr/local/lib/python2.7/dist-packages/absl/app.py", line 299, in run
    _run_main(main, args)
  File "/usr/local/lib/python2.7/dist-packages/absl/app.py", line 250, in _run_main
    sys.exit(main(argv))
  File "main.py", line 583, in main

Hi, How to solve this problem? please help

**Comment by MaybeLL** [comment]
i encouter a same problem when i run the [albert](https://github.com/google-research/ALBERT/blob/master/run_race.py).
I guess that maybe the --train_file and --eval_file occured the problem.
here is my setting:
`--albert_config_file=/home/dy/Project/ALBERT/albert_base/albert_config.json
--output_dir=/home/dy/Project/ALBERT/output_dir
--data_dir=/home/dy/Project/ALBERT
--init_checkpoint=/home/dy/Project/ALBERT/albert_base/model.ckpt-best
--train_file=/home/dy/Project/ALBERT/record_file
--eval_file=/home/dy/Project/ALBERT/record_file
--spm_model_file=/home/dy/Project/ALBERT/albert_base/30k-clean.model
--max_seq_length=512
--max_qa_length=128
--do_train
--do_eval
--train_batch_size=8
--eval_batch_size=8
--learning_rate=1e-5
--train_step=12000
--warmup_step=1000
--save_checkpoints_steps=100`
